.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,16 +1,12 @@
 name: Bug Report
-description: Report a bug or issue with Matplotlib
+description: Report a bug or issue with Matplotlib.
 title: "[Bug]: "
 body:
-  - type: markdown
-    attributes:
-      value: |
-        To help us understand and resolve your issue, please fill out the form to the best of your ability. 
   - type: textarea
     id: summary
     attributes:
       label: Bug summary
-      description: Please provide 1-2 short sentences that succinctly describes the bug
+      description: 1-2 short sentences.
```

```suggestion
      description:  Describe the bug in 1-2 short sentences 
```
think it might be useful to explain what we mean by summary